### PR TITLE
Fixes a bug with the standalone generator.

### PR DESCRIPTION
**Pull request by ceriottm**
Would be good to figure out why the standalone is not autodeployed, while we are at it. @Luthaf ?

### CODE REVIEW

**Comment by Luthaf** [comment]
> Would be good to figure out why the standalone is not autodeployed, while we are at it.

What do you mean? `https://chemiscope.org/chemiscope_standalone.html` exists, and the download link works for me.

**Comment by ceriottm** [comment]
yes, but it's not updated. still downloads the single-panel version.

File: utils/generate_standalone.py [hunk 1]
```diff
@@ -87,6 +87,8 @@ def main():
                         help='chemiscope JSON input that is appended to provide default data')
     parser.add_argument('-o', '--output', type=str, default="",
                         help='output file name. Defaults to chemiscope.html or the name of the JSON')
+    parser.add_argument('-l', '--loader', default=False, action='store_true',
```
Comment (Luthaf) [inline utils/generate_standalone.py:90]: I don't think this flag warrant a short form, I usually prefer command line arguments to be more explicit.

Comment (ceriottm) [reply utils/generate_standalone.py:90]: fine with me

**Pull request merged**